src/tool/cppwinrt/strings/base_coroutine.h

```diff
@@ -0,0 +1,125 @@
+
+#ifndef __clang__
```

Comment:
Clang uses a completely different ABI for coroutine support. This means that the VC <experimental/coroutine> header does not support the Clang frontend.